tests/tests_release_notes_workflow/test_releasenotes_check_functions.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import unittest
+from subprocess import CalledProcessError
+
+from git.git_repository import GitRepository
+from release_notes_workflow.releasenotes_check import ReleaseNotesCheck
+from release_notes_workflow.releasenotes_gitlog_getcommit import GitLogGetCommit
+
+
+class TestReleaseNotesCheck(unittest.TestCase):
+
+    def test_ReleaseNotesCheck(self) -> None:
+        assert ReleaseNotesCheck.check("OpenSearch", "2.2.2", "/tmp/") == "NULL"
+
+    def test_gitlog_getcommitID(self) -> None:
```

Comment:
A test like this actually checks out code and won't work offline. Needs mocks.

Reply:
I have updated the test run against a real git repo db.